common/djangoapps/student/models.py

```diff
@@ -717,6 +717,27 @@ def get_or_create_enrollment(cls, user, course_key):
         return enrollment
 
     @classmethod
+    def get_enrollment(cls, user, course_key):
+        """Returns a CoursewareEnrollment object.
+
+        Args:
+            User : is a Django User object.
+            course_id : is our usual course_id string (e.g. "edX/Test101/2013_Fall)
+
+        Returns:
+            Course enrollment object or None
+        """
+        assert(isinstance(course_key, CourseKey))
```

Comment:
By convention, we generally don't use assertions like this.  If the caller passes us an incorrect type, we assume that a TypeError will be raised.